### ISSUE TITLE
Button: fix formatting issues in the classic theme

### ISSUE THREAD

**Issue by enowak1031** [trimble-oss/modus-wc-2.0#108]
When button is used with variant `filled` and color `warning`, there should be a red outline to avoid accessibility issues.

**Comment by gabriel-piltzer-trimble** [comment]
Fixed
